### PR TITLE
#23 - merged viewHierarchyMixin with View and #25 Torso Cell

### PR DESCRIPTION
**Pull request by kentmw**

### CODE REVIEW

**Comment by mandragorn** [comment]
looks good to me. +2

**Comment by kentmw** [comment]
A bummer, sorry guys, I merged my two pull requests on accident. Another peer review?

File: modules/ServiceCell.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  } else if (typeof exports === 'object') {
+    module.exports = factory(require('./Cell'));
+  } else {
+    root.Torso = root.Torso || {};
+    root.Torso.ServiceCell = factory(root.Torso.Cell);
+  }
+}(this, function(Cell) {
+  'use strict';
+  /**
+   * An non-persistable object that can listen to and emit events like a models.
```
Comment (kentmw) [inline modules/ServiceCell.js]: Give a better description

File: modules/cellPersistenceRemovalMixin.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      if (!this.isModelCompatible) {
+        throw 'Cell does not have sync';
+      }
+    },
+
+    url: function() {
+      if (!this.isModelCompatible) {
+        throw 'Cell does not have url';
+      }
+    },
```
Comment (kentmw) [inline modules/cellPersistenceRemovalMixin.js]: remove comma

**Comment by kentmw** [comment]
+2 from Ariel Wexler (sitting next to me)

**Pull request merged**